modules/baseline-router-performance.adoc

```diff
@@ -22,11 +19,7 @@ on many factors. In particular:
 
 * underlying infrastructure (network/SDN solution, CPU, and so on)
 
-While performance in your specific environment will vary, Red Hat lab
-tests on a public cloud instance of size 4 vCPU/16GB RAM, a single
-HAProxy router handling 100 routes terminated by backends serving
-1kB static pages is able to handle the following number of transactions
-per second.
+While performance in your specific environment will vary, Red Hat lab tests on a public cloud instance of size 4 vCPU/16GB RAM, a single HAProxy router handling 100 routes terminated by backends serving 1kB static pages is able to handle the following number of transactions per second.
```

Comment:
Red Hat _Lab_? Haven't come across this before.
Also, something is off in this sentence. I think you might be able to start a new sentence at, "A single HAProxy...."

Reply:
This content is from the Scalability and Performance team so they do a lot of testing--probably where the labs portion comes from. :)